specs/AppLifecycle/EnvironmentVariables/AppLifecycle - Environment Variables (EV).md

```diff
@@ -310,27 +302,17 @@ packaged, writes to HKCU are virtualized, unless the app has the unvirtualizedRe
 restricted capability and has suppressed registry write virtualization for HKCU.
 
 An unpackaged or packaged Win32 (Desktop Bridge) app can write to HKLM if the
-app is running elevated. A UWP app cannot run elevated. In the normal case,
+app is running elevated. In the normal case,
 where virtualization is not disabled, all writes to HKCU actually go to a
 private hive file similar to:
 %ProgramData%\Packages\<PackageFamilyName>\<Security ID>\SystemAppData\Helium
 
-In a non-Desktop Bridge pure UWP app, using the existing System.Environment
-class, you can get/set at the Process scope only: you can't get or set
-environment variables for User or Machine. The calls are merely no-ops - there's
-no exception, but the return is an empty string. The use of the
-unvirtualizedResources feature in a pure UWP app makes no difference to this.
-However, you can use the managed Registry APIs to read at User scope (but not at
-Machine scope), and p/invoke to the native registry APIs to read at both User
-and Machine scope.
-
 Moving forward, the new APIs will enable the following behaviors:
 
 | App Type       | API Type | Process Vars | User Vars   | Machine Vars |
 | -------------- | -------- | ------------ | ----------- | ------------ |
 | Unpackaged     | Managed  | Read, Write  | Read, Write | Read, Write  |
 | Desktop Bridge | Managed  | Read, Write  | Read, Write | Read, Write  |
-| UWP            | Managed  | Read, Write  | Read        | Read         |
```

Comment:
So UWP is cut? Do you need to mention `windows.partialTrustApplication` instead?

Reply:
The Windows App SDK cut UWP support long ago. Which makes this table pretty useless. I believe @DrusTheAxe mentioned that some time ago but alas the table is still here.

Reply:
Got it. Partial trust is still a thing though so should be mentioned in any discussing of access rights.

Reply:
I don't think WAS supports any partial trust scenarios?

Reply:
We support appcontainers, just not CoreWindow.  UWP is MSIX+AppContainer+Lifecycle+CoreWindow.  I'd _like_ to get us to a point where we can support MSIX+AppContainer+Lifecycle+HWNDs.  The "partial trust application" is MSIX+AppContainer+NoLifecycle+HWNDs, so it's _so close_

Reply:
This API, when used by a partial-trust app, would then only modify its own environment block - rather than the user-wide one.

Reply:
Should I re-add the UWP row and rename it to "Partial-Trust"?